### PR TITLE
Add Case Type to entity and default new case to household.

### PR DESCRIPTION
**Pull request by rjweeks**
# Motivation and Context
Add new Case Type column to Case Processor and Action scheduler cases tables.
This will allow RM to differentiate a Household case from a Household individual PQ case.

# What has changed
Entity has a new CaseType field.

# How to test?
Build new image and execute  in Docker Dev and run acceptance tests.

# Links
https://trello.com/c/gUMjDkBu/1064-create-case-type-variable-to-differentiate-hh-and-hi-as-opposed-to-address-type-5
https://github.com/ONSdigital/census-rm-case-api/pull/26
https://github.com/ONSdigital/census-rm-ddl/pull/6

### CODE REVIEW

**Comment by rjweeks** [comment]
[Create Case Type variable to differentiate HH and HI (as opposed to address type) (5)](https://trello.com/c/gUMjDkBu/1064-create-case-type-variable-to-differentiate-hh-and-hi-as-opposed-to-address-type-5)

File: src/main/java/uk/gov/ons/census/action/messaging/EventReceiver.java [hunk 1]
```diff
@@ -87,6 +88,7 @@ private void setCaseDetails(CollectionCase collectionCase, Case caseDetails) {
     caseDetails.setLongitude(collectionCase.getAddress().getLongitude());
     caseDetails.setUprn(collectionCase.getAddress().getUprn());
     caseDetails.setRegion(collectionCase.getAddress().getRegion());
+    caseDetails.setCaseType("HH");
```
Comment (NickGrantONS) [inline src/main/java/uk/gov/ons/census/action/messaging/EventReceiver.java]: I don't think this should be hard-coded should it?

Comment (rjweeks) [reply src/main/java/uk/gov/ons/census/action/messaging/EventReceiver.java]: Case Type now taken from Case Created Event.

Comment (NickGrantONS) [inline src/main/java/uk/gov/ons/census/action/messaging/EventReceiver.java]: This looks like it's still hard-coded to me

Comment (rjweeks) [reply src/main/java/uk/gov/ons/census/action/messaging/EventReceiver.java]: Fixed

File: src/main/java/uk/gov/ons/census/action/messaging/EventReceiver.java [hunk 2]
```diff
@@ -55,6 +55,7 @@ private void processCaseCreatedEvent(CollectionCase collectionCase) {
     newCase.setCaseId(UUID.fromString(collectionCase.getId()));
     setCaseDetails(collectionCase, newCase);
     newCase.setReceiptReceived(false);
+    newCase.setCaseType(collectionCase.getCaseType());
```
Comment (NickGrantONS) [inline src/main/java/uk/gov/ons/census/action/messaging/EventReceiver.java]: you don't need to do this... it's already done in `setCaseDetails`

Comment (rjweeks) [reply src/main/java/uk/gov/ons/census/action/messaging/EventReceiver.java]: Removed.

File: src/main/java/uk/gov/ons/census/action/messaging/EventReceiver.java [hunk 3]
```diff
@@ -87,6 +87,7 @@ private void setCaseDetails(CollectionCase collectionCase, Case caseDetails) {
     caseDetails.setLongitude(collectionCase.getAddress().getLongitude());
     caseDetails.setUprn(collectionCase.getAddress().getUprn());
     caseDetails.setRegion(collectionCase.getAddress().getRegion());
+    caseDetails.setCaseType(collectionCase.getCaseType());
```
Comment (NickGrantONS) [inline src/main/java/uk/gov/ons/census/action/messaging/EventReceiver.java]: Can you move this to the section below

**Pull request merged**